### PR TITLE
Add support for dry run for `gitlab:group:ensureExists` action.

### PR DESCRIPTION
**Pull request by Andy2003**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by Andy2003** [comment]
Do not close. Still waiting for review.

**Comment by Andy2003** [comment]
Do not close. Still waiting for review.

File: plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabGroupEnsureExistsAction.ts [hunk 1]
```diff
@@ -52,6 +53,11 @@ export const createGitlabGroupEnsureExistsAction = (options: {
       }),
     },
     async handler(ctx) {
+      if (ctx.isDryRun) {
+        ctx.output('groupId', 42);
```
Comment (Rugvip) [inline plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabGroupEnsureExistsAction.ts:57]: Not quite able to formulate a question regarding this specific choice. It'll do 👍

**Review by Rugvip — APPROVED** [review]
Sorry this one flew under my radar, was a simple fix, thank you! 👍

**Comment by albibanushi** [comment]
hey @Andy2003 
I cannot seem to make this plugin work.
Why is `repos` prepended here https://github.com/backstage/backstage/blob/master/plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabGroupEnsureExistsAction.ts#L72 ?

I am having this error even though the group exists
`info: creating missing group repos/xxxx/xxxx/testing` the group full path should be`xxxx/xxxx/testing`

**Comment by Andy2003** [comment]
@albibanushi That was a misinterpretation of our specific setup. I have already created a fix with #18599.

**Pull request merged**